python/paddle/fluid/contrib/layers/metric_op.py

```diff
@@ -185,4 +194,15 @@ def ctr_metric_bundle(input, label):
                 "Y": [local_ins_num]},
         outputs={"Out": [local_ins_num]})
 
+    #if data is fake, return 0
+    ins_tag_weight_np = ins_tag_weight.to_string(False, False)
```

Comment:
if ins_tag_weight is none, it also work?

Reply:
yes, because it have been set default value.